models/chainlink/bnb/chainlink_bnb_schema.yml

```diff
@@ -5,44 +5,39 @@ models:
     meta:
       blockchain: bnb
       project: chainlink
-      contributors: ["msilb7","0xroll"]
+      contributors: ['msilb7','0xroll','linkpool_ryan']
     config:
       tags: ['bnb','chainlink','oracle']
     description: >
         Table pulling price updates for known Chainlink oracle addresses
-    tests:
-      - dbt_utils.unique_combination_of_columns:
-          combination_of_columns:
-              - blockchain
-              - block_number
-              - proxy_address
-              - underlying_token_address
-      - check_seed_legacy:
-          seed_file: ref('chainlink_get_price_seed')
-          filter:
-            blockchain: bnb
-          match_columns:
-            - blockchain
-            - block_number
-            - proxy_address
-            - underlying_token_address
-          check_columns:
-            - aggregator_address
-            - feed_name
-            - oracle_price
+    # Tempoarily Disable, test_data not found.
```

Comment:
we can re-enable this by forcing an update to the seed file (`chainlink_get_price_seed.sql`), so it pushes to this PR.

then for the columns listed, both match and check, you can create a schema file in the seeds directory path which calls out their datatype and applies a `tag:dunesql`. here is an example: 
https://github.com/duneanalytics/spellbook/blob/main/seeds/_sector/nft/_schema.yml#L1-L15

Reply:
you only need to add columns which leverage unique dunesql data types typically (varbinary, uint256, etc)